features/step_definitions/rails_steps.rb

```diff
@@ -1,3 +1,14 @@
+When /^I create a new rails application$/ do
+  options = "--skip-bootsnap --skip-javascript"
+  template = "-m ../../features/support/rails_template"
+  result = run_command("bundle exec rails new test_app #{options} #{template}")
```

Comment:
What you think about creating an app in API mode? (`.. --api`) it will skip generating views, helpers, and assets. I think it might increase tests speed 🤔

Reply:
I also think about:
```
--skip-action-mailer
--skip-active-storage
--skip-action-cable
```
for example, I might be wrong, but I think it might be helpful :)

Reply:
Yup, I think that is a good idea. The tests already got a bit faster by skipping js. And the fewer dependencies we have the less flaky these tests become (many of our failures are a result of some incompatible version of some gem).

I was also thinking it would be nice if we didn't need to install the development dependencies, since we don't ever start the app in development mode. Perhaps we could also `--skip bundle` and then manually run `bundle install --without development`.

I am going to merge this as is for now to get a passing build on master, and then open a separate issue to explore reducing the number of gems we need to install.

Reply:
Sounds good let's do it! 👍

Reply:
I opened #370